apps/web/stores/NavigationStore.ts

```diff
@@ -143,4 +157,58 @@ export class NavigationStore {
       this.currentItemId,
     );
   };
+
+  itemIsComplete = (itemId: string) => {
+    return (
+      this.rootStore.answerStore.answers[itemId] !== undefined &&
+      (this.currentItemId !== itemId ||
+        (this.currentItemId === itemId && !this.nextButtonIsDisabled))
+    );
+  };
+
+  itemWasSkipped = (
```

Comment:
I think some comments on these for `itemWasSkipped`, `pastSection`, and `sectionIsComplete` would be helpful. Like a higher level "...checks if current item aka question is behind the farthest question answered; then it was skipped...". Something along those lines for each.

Reply:
Added some comments! LMK what you think.